packages/design-system/src/utils/functions.ts

```diff
@@ -6,3 +6,6 @@ export const openLink = (link: string, newTab = false) => {
   window.location.href = link
   return
 }
+
+export const isLast = (index: number, array: any[]) =>
+  index === array.length - 1
```

Comment:
I'd suggest to avoid `any` as hard as you can. You'd better use `unknown` or generics:

```suggestion
export const isLast = (index: number, array: unknown[]) =>
  index === array.length - 1
```

```suggestion
export const isLast = <T>(index: number, array: T[]) =>
  index === array.length - 1
```